README.md

```diff
@@ -69,6 +69,13 @@ Fedora 25. Other nodesets are currently not present. The module requires at
 least Puppet 4.9.4 (this is the first version with proper data-in-modules
 support).
 
+## Drawbacks
```

Comment:
I might have gone with ‘Limitations’, but otherwise good information for CentOS users. RedHat users can probably guess it also applies to them!

Reply:
fixed